dev-docs/bidders/admatic.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: bidder
+title: AdMatic
+description: Prebid AdMatica Bidder Adapter.
+pbjs: true
+biddercode: admatic
+media_types: banner,video
+---
+
+### Description
+
+One of the easiest way to gain access to AdMatic demand sources  - AdMatic header bidding adapter.
```

Comment:
```suggestion
```

The docs should contain technical information and no marketing phrases to keep them concise and readable :wink: